packages/hdwallet-portis/src/portis.ts

```diff
@@ -82,11 +203,90 @@ export function isPortis(wallet: HDWallet): wallet is PortisHDWallet {
   return typeof wallet === 'object' && (wallet as any)._isPortis === true
 }
 
-export class PortisHDWallet implements HDWallet, ETHWallet {
+export class PortisHDWallet implements HDWallet, ETHWallet, BTCWallet {
+
+  // btc stuff
+
+
+  public async btcGetAddress (msg: BTCGetAddress): Promise<string> {
+    
+    const change = msg.addressNList[3]
```

Comment:
Don't assume any structure on the `addressNList` passed in.

Reply:
also don't assume `addressNList.length > 0`. If you need to assume so, throw an error in those cases.... but I'd prefer to make this as generic as possible, agnostic to paths.